### PR TITLE
Added basic README

### PR DESCRIPTION
**Pull request by Laumania**
I guess other than me have doubts about what is needed for contributing
to this project, so guess the README would be a good place to put in
various prerequisites + links.

I don't know whats needed, so if you could provide I could update README before merging in the PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## JabbR - v2
+JabbR is a chat application built with ASP.NET using SignalR.
+
+## Prerequisites
+To be able to contribute to the project, you need to following new bits:
+* Visual Studio 2014 - Link ?
```
Comment (hnrkndrssn) [inline README.md]: That should be VS2015 right? Link: http://www.visualstudio.com/en-us/downloads/visual-studio-2015-downloads-vs.aspx

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+* .NET vNext - Link?
```
Comment (hnrkndrssn) [inline README.md]: Link: https://github.com/aspnet/Home? :)

**Comment by Laumania** [comment]
Now updated with links, but do i miss more?

**Comment by Laumania** [comment]
@davidfowl Can you confirm that these are the correct prerequisites for developing on JabbR v2?
Besides that, I want to put into the readme what the purpose of JabbR v2 is. We know what JabbR is, but why this V2 :)

**Comment by davidfowl** [comment]
Orrr OMNISHARP!

**Comment by Laumania** [comment]
Omnisharp as an alternative to VS2015?

**Comment by hnrkndrssn** [comment]
That's true! (not all of us will be on Windows and be able to _use_ VS2015 or _choose_ to use something else ;))

**Comment by Laumania** [comment]
OmniSharp has been added :)
More?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## JabbR - v2
+JabbR is a chat application built with ASP.NET using SignalR.
```
Comment (hnrkndrssn) [inline README.md]: Sorry, missed this last time around, that should be ASP.NET 5

File: README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+## Prerequisites
+To be able to contribute to the project, you need to following new bits:
+* [Visual Studio 2015](http://www.visualstudio.com/en-us/downloads/visual-studio-2015-downloads-vs.aspx) or alternatively [OmniSharp](http://www.omnisharp.net/)
+* .NET vNext - [Get is here](https://github.com/aspnet/home#getting-started)
```
Comment (hnrkndrssn) [inline README.md]: Minor spelling error, should be Get _**it**_ here

Comment (Laumania) [inline README.md]: Thanks for your feedback, will fix it soon :)

**Comment by Laumania** [comment]
Added a Getting Started part, which I'm totally unsure are correct. This is how I made the app run on my machine, but this is my first encounter with ASP.NET 5 and all this "new way of doing things", so I'm not completely sure what I'm doing and why :)

**Comment by Laumania** [comment]
Lets just close this one, as you have added a README now - and a much better one than my version :P

**Comment by hnrkndrssn** [comment]
Sorry about that :innocent:

**Comment by Laumania** [comment]
No problem :)

**Pull request closed without merging**